donkeycar/parts/dgym.py

```diff
@@ -48,34 +48,33 @@ def __init__(self, cfg, outputs):
 
         # output keys corresponding to info dict values
         self.info_keys = {
```

Comment:
Because this dict has now the structure { k: v } with v == k, we can simply turn it into a list of keys only [k].

Reply:
I see we can't there is still a map from `odom` to something else.